### ISSUE TITLE
Available use of leaflet event

### ISSUE THREAD

**Issue by bthieurmel** [rstudio/leaflet#37]
Hi,
Is there a way to use leaflet event via the package or do you think about integrate that in the futur ?
It can be very usefull !

**Comment by jcheng5** [comment]
Can you tell us a little more about what you're trying to accomplish? It's possible we're already exposing the events you want. Thanks!

**Comment by bthieurmel** [comment]
Yes of course ! I want for example can use the zoomend methods for enabled or not some data per zoom level
```
map.on('zoomend', onZoomend);
function onZoomend(){
...
}
```

**Comment by jcheng5** [comment]
You can do this if you're using it with Shiny. If your leafletOutput has an output ID of "foo" then you can refer to `input$foo_zoom`. There's also `input$foo_bounds` which is a named list with `north`/`south`/`east`/`west` elements.

**Comment by bthieurmel** [comment]
Ok, thanks. And so, if I understand, there is no way actually to do this using in the viewer or saving to html file. Do you plan to explore this features or not ?

**Comment by pssguy** [comment]
@jcheng5 

How would I do this in rmarkdown shiny?  I use the  `zoomToLimits="first" `option but would like the initial map
to be zoomed in 1 or 2 steps (it currently shows around 3 sets of the world)

    output$map <- renderLeaflet({

    leaflet( blah blah blah) %>% 
    mapOptions(zoomToLimits="first")

    })

    leafletOutput("map")
